assets/js/components/Project/Form.jsx

```diff
@@ -31,12 +31,41 @@ const ProjectForm = ({ showForm, setShowForm }) => {
     }
   );
 
+  let [
+    updateProject,
+    {
+      loading: projectUpdateLoading,
+      error: projectUpdateError,
+      projectUpdateData,
+    },
+  ] = useMutation(UPDATE_PROJECT, {
+    onCompleted({ updateProject }) {
+      toastWrapper(
+        "is-success",
+        `Project ${updateProject.title} updated successfully`
+      );
+      setShowForm(false);
+    },
+    onError(error) {
+      setFormError(error);
+    },
+    refetchQueries(mutationResult) {
+      return [{ query: GET_PROJECTS }];
+    },
```

Comment:
Pretty sure as long as the `query` returns and item's `id`, we don't need to refetch the Queries.  It should update the Apollo cache automagically.